scripts/ironbank/Dockerfile

```diff
@@ -0,0 +1,93 @@
+    microdnf install --assumeyes \
+      ca-certificates \
+      git \
+      gzip \
+      shadow-utils \
+      tar \
+      unzip && \
+    microdnf clean all
+
+# Configure the cryptography policy manually. These policies likely
+# have no impact, since Go doesn't link against these libraries.
+#
+# Normally, one uses the update-crypto-policies script to create these
+# links, which is included in the crypto-policies-scripts package, but
+# that pulls in Python, so we create the links manually here. This
+# list of links comes from running strace on the update-crypto-policies
+# script (strace update-crypto-policies --set FIPS) in Fedora, since
+# RHEL and UBI do not provide an strace package by default.
+RUN echo "FIPS" >/etc/crypto-policies/config && \
+    cp --force /usr/share/crypto-policies/policies/FIPS.pol /etc/crypto-policies/state/CURRENT.pol && \
+    echo "FIPS" >/etc/crypto-policies/state/current && \
+    ln --symbolic --force /usr/share/crypto-policies/FIPS/bind.txt /etc/crypto-policies/back-ends/bind.config && \
+    ln --symbolic --force /usr/share/crypto-policies/FIPS/gnutls.txt /etc/crypto-policies/back-ends/gnutls.config && \
+    ln --symbolic --force /usr/share/crypto-policies/FIPS/java.txt /etc/crypto-policies/back-ends/java.config && \
+    ln --symbolic --force /usr/share/crypto-policies/FIPS/krb5.txt /etc/crypto-policies/back-ends/krb5.config && \
+    ln --symbolic --force /usr/share/crypto-policies/FIPS/libreswan.txt /etc/crypto-policies/back-ends/libreswan.config && \
+    ln --symbolic --force /usr/share/crypto-policies/FIPS/libssh.txt /etc/crypto-policies/back-ends/libssh.config && \
+    ln --symbolic --force /usr/share/crypto-policies/FIPS/nss.txt /etc/crypto-policies/back-ends/nss.config && \
+    ln --symbolic --force /usr/share/crypto-policies/FIPS/openssh.txt /etc/crypto-policies/back-ends/openssh.config && \
+    ln --symbolic --force /usr/share/crypto-policies/FIPS/opensshserver.txt /etc/crypto-policies/back-ends/opensshserver.config && \
+    ln --symbolic --force /usr/share/crypto-policies/FIPS/openssl.txt /etc/crypto-policies/back-ends/openssl.config && \
+    ln --symbolic --force /usr/share/crypto-policies/FIPS/opensslcnf.txt /etc/crypto-policies/back-ends/opensslcnf.config
+
+# Copy and extract Coder binary from tar file.
+ARG CODER_BIN=/opt/coder
+RUN mkdir -p /opt
+ARG CODER_BIN_TAR_GZ=coder.tar.gz
+COPY "$CODER_BIN_TAR_GZ" /tmp/coder.tar.gz
+RUN tar -xzvf /tmp/coder.tar.gz --directory /opt --strip-components=1 ./coder && \
+    rm /tmp/coder.tar.gz
+
+# Copy and extract Terraform binary from zip file.
+ARG TERRAFORM_BIN_DIR=/opt/terraform
+RUN mkdir -p "$TERRAFORM_BIN_DIR"
+ARG TERRAFORM_BIN_ZIP=terraform.zip
+COPY "$TERRAFORM_BIN_ZIP" /tmp/terraform.zip
+RUN unzip /tmp/terraform.zip -d "$CODER_BIN_DIR" && \
+	  rm /tmp/terraform.zip
+ENV PATH="${TERRAFORM_BIN_DIR}:${PATH}"
+
+# Install the Coder Terraform provider to a well-known location.
+ARG TERRAFORM_PLUGINS_DIR=/opt/terraform/plugins
```

Comment:
Do we have to install any other providers for them?

Reply:
Maybe Kubernetes or something?

Reply:
nope - any other Terraform providers they need is up to them. they can install those providers from a self-hosted provider registry.

Reply:
Coolio